### DISCUSSION TITLE
OCP v4.17 is out. Is there still a future for OKD-SCOS v4.16? or will the next OKD GA will be v4.17?

### DISCUSSION THREAD

**Discussion by titou10titou10** [okd-project/okd#2040] [Q&A] [answered]
OCP v4.17 is out: https://www.reddit.com/r/openshift/comments/1ftdsxv/whats_new_in_openshift_417_key_updates_and_new/

Q:
- will the next GA version of OKD-SCOS will be v4.16?
- or will v4.16 be skipped in favour of v4.17?
- if so, will there be a path to migrate from OKD-FCOS v4.15 to OKD-SCOS v4.17?

Thx for your work!

**Comment by JaimeMagiera** [comment, marked as answer]
The GA OKD-SCOS will be 4.16. It will be considered a step update to get installs from FCOS to SCOS. From there, users can then update to 4.17.
